discovery-infra/delete_nodes.py

```diff
@@ -98,5 +106,33 @@ def main():
         type=str,
         default="assisted-installer",
     )
+    parser.add_argument(
```

Comment:
looks like this is a time to change usage and to add subcommand. Can u please open a ticket?
we must have single entrypoint and subcommands as install, delete ...